test/filters.js

```diff
@@ -26,16 +26,23 @@ const file = new Vinyl({
   contents: fs.readFileSync(filePath),
 });
 
-test(
-  'should compile a pug template with a custom pug instance with filters',
-  function(t) {
-    const stream = task(options);
-    stream.on('data', function(newFile) {
-      t.ok(newFile);
-      t.ok(newFile.contents);
-      t.equal(newFile.contents.toString(), 'HELLO, TESTER!!!!');
-      t.end();
-    });
-    stream.write(file);
-  }
-);
+describe('filters', function() {
+  it(
+    'should compile a pug template with a custom pug instance with filters',
```

Comment:
If you want to add a `.eslintrc` into the `test` directory that extends `"gulp/test"`, you could put the `it`, description and `function` all on 1 line.

Reply:
Fixed